shared.gpr

```diff
@@ -138,11 +138,10 @@ abstract project Shared is
      & Global_Options & Instrument_Options;
    --  Common options used for the Debug and Release modes
 
-   Debug_Options :=
-     ("-g", "-gnata", "-gnatVa", "-gnatQ", "-gnato", "-gnatwe", "-Wall");
+   Debug_Warnings := ("-g", "-gnata", "-gnatVa", "-gnatQ", "-gnato", "-Wall");
+   Debug_Options  := Debug_Warnings & "-gnatwe";
```

Comment:
Instead of splitting in two options I think you could have inhibited the -gnatwe option for ssl-thin.ads below.

Using -gnat-N or -gnat-m (only line length disabled).

Reply:
I increased the max line length for ssl-thin.ads.